### PR TITLE
Add lb/mil² and Wh/lbs

### PR DESCRIPTION
**Pull request by rutemja**
Adds PoundForcePerSquareMil (see #1091) and WattHourPerPound (see #1090).

### CODE REVIEW

File: UnitsNet.Tests/CustomCode/SpecificEnergyTests.cs [hunk 1]
```diff
@@ -22,6 +22,11 @@ public class SpecificEnergyTests : SpecificEnergyTestsBase
         protected override double MegawattHoursPerKilogramInOneJoulePerKilogram => 2.77777778e-10;
         protected override double GigawattHoursPerKilogramInOneJoulePerKilogram => 2.77777778e-13;
 
+        protected override double WattHoursPerPoundInOneJoulePerKilogram => 1.2599788055556e-4;
```
Comment (angularsen) [inline UnitsNet.Tests/CustomCode/SpecificEnergyTests.cs:25]: [Verified](https://www.google.com/search?q=WattHoursPerPoundInOneJoulePerKilogram&rlz=1C1ONGR_enNO951NO951&sxsrf=ALiCzsYBjlTxTzOMF8qLtZjlsHMbGTq00Q%3A1653937064169&ei=qBOVYor0CYr_kwWC_YjACg&ved=0ahUKEwiK6MLZ84f4AhWK_6QKHYI-AqgQ4dUDCA4&uact=5&oq=WattHoursPerPoundInOneJoulePerKilogram&gs_lcp=Cgdnd3Mtd2l6EAM6BwgAEEcQsANKBAhBGABKBAhGGABQ6QFY6QFgwQdoAXABeACAAbEBiAGxAZIBAzAuMZgBAKABAqABAcgBCMABAQ&sclient=gws-wiz)

**Review by angularsen — APPROVED** [review]
Looks perfect to me, great work!

**Comment by angularsen** [comment]
Nuget should be out in a few minutes.

[Release UnitsNet/4.132.0 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F4.132.0)

**Comment by rutemja** [comment]
Thanks :)

**Pull request merged**